server/README.md

```diff
@@ -0,0 +1,16 @@
+#  
+First launch:
+    Make sure you have docker and postgres installed.
+    Open terminal:
+    Type:
+    docker run --name postgres-db -e POSTGRES_PASSWORD=password -p 25432:5432 -d postgres
+Second launch:
+    Open terminal:
+    Type:
+    docker start postgres-db
+
+To open database:
```

Comment:
Generalnie nie ma czegoś takiego jak otwieranie bazy danych. Jeśli już, jest to łączenie się z nią.
No i takich rzeczy nie pisze się w README, one nie dotyczą uruchamiania aplikacji.